config/config.go

```diff
@@ -47,6 +48,69 @@ func (c *Config) Scheme() string {
 	}
 }
 
+func CiphersuiteByName(name string) uint16 {
+	switch name {
+	case "TLS_RSA_WITH_RC4_128_SHA":
+		return tls.TLS_RSA_WITH_RC4_128_SHA
```

Comment:
RC4 and 3DES are not recommended to use now. Do you have any good reason to support them?

Reply:
Thank you for review!

For some reason Golang didn't provide literal names for ciphersuites and it wasn't possible to get ciphersuite id by name. So I had to add a such convert function on my own. I've listed all TLS1.0-1.2 ciphersuites here without any security considerations just to keep the function more library-like. TLS 1.3 ciphersuites are not listed there since they are not configurable. That's why I've covered all ciphersuites [listed](https://github.com/golang/go/blob/c170b14c2c1cfb2fd853a37add92a82fd6eb4318/src/crypto/tls/cipher_suites.go#L500) in `crtypto/tls` module.

But I actually was surprised that master branch of Golang was updated after this PR was created. In upcoming 1.14 release they have added a `CipherSuite()` [function](https://github.com/golang/go/blob/c170b14c2c1cfb2fd853a37add92a82fd6eb4318/src/crypto/tls/cipher_suites.go#L52), which provide the missing link between literal names and ids which can be used to get ciphersuite id by literal name (simillary to new `cipherSuiteByID()` [function](https://github.com/golang/go/blob/c170b14c2c1cfb2fd853a37add92a82fd6eb4318/src/crypto/tls/cipher_suites.go#L468)). 

I can adapt PR to changes in upcoming 1.4 Golang release, but then I will need to bump `go` version in `go.mod` and the PR will have to be postponed until Golang 1.4 is released (it's in beta now).
Or I can leave the changes as is, then the code will work fine in both Go 1.2 and 1.4. Which variant do you prefer?

Reply:
Thank you for describing! 
It's OK to use your current code because Go 1.4 has not released yet.